micheline/type_test.go

```diff
@@ -520,10 +518,22 @@ func detectBigmapTypes(p Prim) map[string]Type {
 		if n == "" {
 			n = "bigmap_" + strconv.Itoa(i)
 		}
-		if _, ok := named[n]; ok {
-			n += "_" + strconv.Itoa(i)
+		uniqueName := func(n string) string {
```

Comment:
Updated this just to match the naming convention defined in `BigmapTypeDetection`. Otherwise, tests would fail due to a mismatch in naming.